editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/wizard/Wsdl2RestWizard.java

```diff
@@ -0,0 +1,359 @@
+					tool.setJaxwsAddress(targetAddressURL);
+				}
+				if (!Strings.isEmpty(options.getTargetRestServiceAddress())) {
+					URL targetRestAddressURL = new URL(options.getTargetRestServiceAddress());
+					tool.setJaxrsAddress(targetRestAddressURL);
+				}
+				if (outJavaPath != null && outJavaPath.toFile().exists()) {
+					tool.setJavaOut(outJavaPath);
+				}
+				ClassLoader loader = tool.getClass().getClassLoader();
+				Thread.currentThread().setContextClassLoader(loader);
+				tool.process();
+				updateDependencies();
+			}  finally {
+				Thread.currentThread().setContextClassLoader(oldLoader);
+				project.refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());
+			}
+		}
+	}
+
+	/**
+	 * @param isBlueprint
+	 * @param isSpringBoot
+	 * @return
+	 */
+	public String getCamelFileName(boolean isBlueprint, boolean isSpringBoot) {
+		if (isBlueprint) {
+			return DEFAULT_BLUEPRINT_CONFIG_NAME;
+		}
+		if (isSpringBoot) {
+			return DEFAULT_SPRINGBOOT_CONFIG_NAME;
+		}
+		return DEFAULT_CONFIG_NAME;
+	}
+	
+	/**
+	 * @param isBlueprint
+	 * @param camelFile
+	 * @param tool
+	 */
+	protected Path initContextForTool(boolean isBlueprint, boolean isSpringBoot, File camelFile, Wsdl2Rest tool) {
+		Path contextpath = null;
+		if (camelFile != null && !camelFile.isDirectory()) {
+			contextpath = new File(camelFile.getAbsolutePath()).toPath();
+			tool.setCamelContext(contextpath);
+		} else {
+			String camelConfigName = getCamelFileName(isBlueprint, isSpringBoot);
+			if (camelFile != null) {
+				contextpath = new File(camelFile.getAbsolutePath() + File.separator + camelConfigName).toPath();
+				tool.setCamelContext(contextpath); 
+			} else {
+				IProject project = options.getProject();
+				IPath projectPath = ResourcesPlugin.getWorkspace().getRoot().findMember(project.getName()).getLocation();
+				contextpath = new File(projectPath.makeAbsolute() + File.separator + camelConfigName).toPath();
+				tool.setCamelContext(contextpath); 
+			}
+		}
+		return contextpath;
+	}
+
+	/**
+	 * @param resource
+	 * @return
+	 * @throws CoreException
+	 */
+	protected File findFileInEFS(IResource resource) throws CoreException {
+		File fileFound = null;
+
+		// gets URI for EFS.
+		URI uri = resource.getLocationURI();
+
+		// what if file is a link, resolve it.
+		if(resource.isLinked()){
+			uri = resource.getRawLocationURI();
+		}
+
+		// Gets native File using EFS
+		fileFound = EFS.getStore(uri).toLocalFile(0, new NullProgressMonitor());			
+
+		return fileFound;
+	}
+
+	/**
+	 * Returns the shared Wsdl2RestOptions object 
+	 * @return Wsdl2RestOptions
+	 */
+	public Wsdl2RestOptions getOptions() {
+		return options;
+	}
+
+	@Override
+	public boolean canFinish() {
+		boolean flag = super.canFinish();
+		if (flag && !this.inTest && 
+				getContainer().getCurrentPage().equals(this.getStartingPage()) && 
+				pageTwo.getMessageType() == IMessageProvider.WARNING &&
+				pageTwo.getMessage().startsWith("Caution:")) { //$NON-NLS-1$
```

Comment:
is it "Caution:" also if someone is using another language?
Where does this "Caution" coming from?

Reply:
Caution comes from https://github.com/jbosstools/jbosstools-fuse/pull/1362/files#diff-9afd4b5273a020f133f09dce67f592d0 in one of the two "Caution:" messages.  It was an easy way of flagging a warning that didn't prevent the user from stopping the process, but does make them go to the next page in the wizard so they SEE those warnings. 

I'm open to suggestions, but this was a low tech solution. 

Considering that I don't know that we have actually localized anything though we have the facility to do so, I wasn't too concerned with what "Caution" might be in other languages.

Reply:
Created https://issues.jboss.org/browse/FUSETOOLS-2970